subsys/net/ip/net_if.c

```diff
@@ -163,7 +164,7 @@ static bool net_if_tx(struct net_if *iface, struct net_pkt *pkt)
 
 		net_pkt_unref(pkt);
 	} else {
-		net_stats_update_bytes_sent(pkt_len);
+		net_stats_update_bytes_sent(pkt->total_pkt_len);
```

Comment:
I am missing something here but where is pkt->total_pkt_len set?

Reply:
It is set in net_if_queue_tx() in line 201

Reply:
in next patch right, not this one?

Reply:
> in next patch right, not this one?

Yes, I merged the two statistics patches to one.